perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/memberstabs/SendPasswordResetRequestTabItem.java

```diff
@@ -134,7 +157,8 @@ public void onChange(ChangeEvent event) {
 			@Override
 			public void onClick(ClickEvent clickEvent) {
 				SendPasswordResetLinkEmail request = new SendPasswordResetLinkEmail(JsonCallbackEvents.closeTabDisableButtonEvents(changeButton, tab));
-				request.sendEmail(member, namespaces.getItemText(namespaces.getSelectedIndex()).toLowerCase());
+				request.sendEmail(member, namespaces.getItemText(namespaces.getSelectedIndex()).toLowerCase(),
+						urns.get(emails.getItemText(emails.getSelectedIndex())), languages.getItemText(languages.getSelectedIndex()));
```

Comment:
Once you modify the box to have different displayed and backend value, you must use ``emails.getItemValue()`` or something like this instead of getItemText() which I believe returns only displayed value.